content/inside-rust/call-for-testing-hint-mostly-unused.md

```diff
@@ -0,0 +1,207 @@
++++
+path = "inside-rust/2025/07/14/call-for-testing-hint-mostly-unused"
+title = "Call for Testing: Speeding up compilation with `hint-mostly-unused`"
+authors = ["Josh Triplett"]
++++
+
+I'm pleased to announce, and call for testing of, the nightly-only `rustc`
+`-Zhint-mostly-unused` option, and the corresponding nightly Cargo features
+`profile.hint-mostly-unused` and `hints.mostly-unused`. These options can help
+accelerate your Rust compile time in some cases, by avoiding compilation of
```

Comment:
"by avoiding compilation" may be interpreted as saying we will skip type-checking and/or borrow-checking. Which is not true in the current implementation. But maybe we want to leave open the possibility of doing so in the future?

I don't have a particular suggestion or concern here, I just want to make sure you mean the implication I'm reading.

Reply:
I definitely didn't intend that interpretation, no. I've rephrased accordingly.